backend/sirius-web-compatibility/src/main/java/org/eclipse/sirius/web/compat/utils/BeanUtil.java

```diff
@@ -0,0 +1,28 @@
+package org.eclipse.sirius.web.compat.utils;
+
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Service;
+
+/**
+ * Utility class used to get bean when outside of spring context.
+ *
+ * @author Charles Wu
+ */
+@Service
+public class BeanUtil implements ApplicationContextAware {
+
+    private static ApplicationContext context;
+
+    @Override
+    public void setApplicationContext(@NonNull ApplicationContext applicationContext) throws BeansException {
+        context = applicationContext;
+    }
+
+    public static <T> T getBean(Class<T> beanClass) {
+        return context.getBean(beanClass);
+    }
+
+}
```

Comment:
No need to use such class, use the constructor-based dependency injection from Spring